catalog/spatial/wfs/spatial-wfs-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/common/FeatureAttributeDescriptor.java

```diff
@@ -60,4 +62,18 @@ public FeatureAttributeDescriptor(AttributeDescriptor ad) {
   public String getPropertyName() {
     return propertyName;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
❓ Why do we need to override `equals`/`hashcode` ? Same question in all the other instances that we're overriding.

Reply:
I intended on making things more explicit and, of course, it stopped SonarCloud's finding.

Reply:
I think if relying on the super class implementation is what we want, we should just mark that SonarCloud finding as a non-issue